### PR TITLE
add feature supporting axis' type to be int in paddle.flip function

### PR DESCRIPTION
**Pull request by sljlp**
### PR types

New features
### PR changes

APIs
### Describe

add a feature supporting axis's type to be int in paddle.flip

### CODE REVIEW

**Review by Superjomn — DISMISSED** [review]
LGTM

**Review by Shixiaowei02 — DISMISSED** [review]

**Review by wangxicoding — APPROVED** [review]
LGTM

File: python/paddle/tensor/manipulation.py [hunk 1]
```diff
@@ -250,6 +250,8 @@ def flip(x, axis, name=None):
     check_dtype(dtype, 'X',
                 ['float16', 'float32', 'float64', 'int32', 'int64', 'bool'],
                 'flip')
+    if isinstance(axis, int):
```
Comment (wangxicoding) [inline python/paddle/tensor/manipulation.py]: 文档部分后面更新一下

**Review by wangxicoding — DISMISSED** [review]

File: python/paddle/tensor/manipulation.py (hunk 1 above)
Comment (sljlp) [reply python/paddle/tensor/manipulation.py]: ok

**Pull request closed without merging**